src/com/connectsdk/service/capability/MediaPlayer.java

```diff
@@ -82,6 +84,10 @@
 
     public void playMedia(MediaInfo mediaInfo, boolean shouldLoop, LaunchListener listener);
 
+    public void playMedia(MediaInfo mediaInfo, boolean shouldLoop, long startPosition, LaunchListener listener);
+
+    public void playMedia(MediaInfo mediaInfo, boolean shouldLoop, long startPosition, Object customData, LaunchListener listener);
```

Comment:
Why `Object customData`? It isn't used anywhere. And the `MediaInfo` object already has the customData attribute now?